specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/virtualNetworkGateway.json

```diff
@@ -2459,6 +2459,24 @@
       ],
       "description": "VPN client revoked certificate of virtual network gateway."
     },
+    "RadiusServer": {
+      "properties": {
+        "radiusServerAddress": {
+          "type": "string",
+          "description": "The address of this radius server."
+        },
+        "radiusServerScore": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The initial score assigned to this radius server."
+        },
+        "radiusServerSecret": {
```

Comment:
>"radiusServerSecret" [](start = 8, length = 20)

Secrets should not be exposed in GETs or as the responses to PUT/PATCH.  They should only be exposed in POST calls (but can be set via PUT/PATCH as long as they don't show up in the response).  How is this secret handled in the API?

Reply:
This secret is not exposed. Only set through put

Reply:
The examples should not return the property at all, right now they show an empty string.  Also, please add x-ms-mutability to make it clear to consumers when/where this property will show up  https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-mutability